bukuserver/views.py

```diff
@@ -117,7 +117,7 @@ def _create_ajax_loader(self, name, options):
     def _list_entry(self, context: Any, model: Namespace, name: str) -> Markup:
         LOG.debug("context: %s, name: %s", context, name)
         parsed_url = urlparse(model.url)
-        netloc = parsed_url.netloc
+        netloc = buku.get_netloc(model.url) or ''
```

Comment:
…Since we have a dedicated function for obtaining netloc, I thought we might as well use it in webUI as well